src/main/java/io/nats/client/api/ConsumerConfiguration.java

```diff
@@ -366,7 +366,7 @@ public Builder(ConsumerConfiguration cc) {
          * @return the builder
          */
         public Builder description(String description) {
-            this.description = validateMaxLength(description, 4096, false, "Description");
+            this.description = validateDescription(description, false);
```

Comment:
I would not validate the description - server allowed length etc should be left to the server.

Reply:
@ripienaar @kozlovic @ColinSullivan1 @wallyqs Can we get a consensus on whether the client should pre-validate the description length. I wonder if we should build an adr (or add and re-purpose the naming conventions doc ) on what things should be client validated and what should be left for the server.

Reply:
Agree with Alberto that client should not validate this. Server will do. One less thing to do in the client that also possibly would lead to "breaking changes" if server wanted to lower (or increase) this limit.

Reply:
ok. validations removed